server/runtime/pom.xml

```diff
@@ -136,10 +136,15 @@
          <artifactId>metainf-services</artifactId>
       </dependency>
       <dependency>
-         <groupId>io.netty</groupId>
-         <artifactId>netty-tcnative</artifactId>
-         <classifier>linux-x86_64</classifier>
+         <groupId>org.wildfly.openssl</groupId>
+         <artifactId>wildfly-openssl-java</artifactId>
       </dependency>
+      <dependency>
+         <groupId>org.wildfly.openssl</groupId>
+         <artifactId>wildfly-openssl-linux-x86_64</artifactId>
+         <scope>compile</scope>
```

Comment:
`compile` is the default scope. 
aren't we going to ship any other natives? macos, windows?

Reply:
no, Linux x86_64 only.